source/braille.py

```diff
@@ -1996,7 +2076,8 @@ def setDisplayByName(  # noqa: C901
 			oldDisplay = self.display
 			if detected and bdDetect._isDebug():
 				log.debug("Possibly detected display '%s'" % newDisplay.description)
-			if newDisplay == oldDisplay.__class__:
+			sameDisplayReinit = newDisplay == oldDisplay.__class__

```

Comment:
Capitalize the first I in ReInit

```suggestion
			sameDisplayReInit = newDisplay == oldDisplay.__class__
```